configuration/pih/petl/jobs/data_kind/encounter/source.sql

```diff
@@ -0,0 +1,14 @@
+SELECT 
+    patient_id,
+    encounter_id,
+    encounter_type,
```

Comment:
encounter_type in the encounter table is just an int (foreign keying to the encounter_type table)... don't we want some method like ENCOUNTER_TYPE_NAME used here to resolve the actual name of the encounter_type?

Reply:
fixed